docs/topics/run-code-snippets.md

```diff
@@ -55,30 +55,135 @@ To rebuild the module automatically before each run of a scratch or a worksheet,
 
 ![Scratch select module](scratch-select-module.png){width=700}
 
-### Running as REPL 
+### Run as REPL 
 
-To evaluate each particular expression in a scratch or a worksheet, run it with **Use REPL** selected. The code will be 
-executed the same way as in [REPL](#repl): the code lines will run sequentially, providing results of each call. 
+To evaluate each particular expression in a scratch or a worksheet, run it with **Use REPL** selected. The code lines
```

Comment:
@yalishevant Do you remember the results of our discussions about «Run as REPL» feature? Do we plan to deprecate it?

Reply:
Yes, we _decided_ to remove this check-box. Run as REPL will be the only way of running.
We heaven't planned it for a particular time and person who will do this. Roma Golyshev is now estimating the scope of work.

Reply:
So we're leaving it as is, I guess? I'll keep in mind the future update. Could you share the corresponding YT issue or add me to watchers?